cluster/gce/upgrade.sh

```diff
@@ -162,7 +190,9 @@ if [[ "${master_upgrade}" == "false" ]] && [[ "${node_upgrade}" == "false" ]]; t
   exit 1
 fi
 
-tars_from_version ${1}
```

Comment:
Why not handle the entire `upload-server-tars` flow here, rather than doing the convuluted `local_binaries_uploaded` check?